packages/asset-uploader/package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.131.0",
     "@aws-sdk/lib-storage": "^3.131.0",
+    "@types/node": "^18.7.16",
```

Comment:
There's already `@types/node` in devDependencies

Reply:
good catch